projects/common/02drv_sensors/02drv_sensors.c

```diff
@@ -0,0 +1,64 @@
+/**
+\brief This is a program which shows how to use the "opensensors" driver module.
+
+Since the driver modules for different platforms have the same declaration, you
+can use this project with any platform.
+
+This application allows you to verify the correct functioning of the opentimers
+drivers. It starts 3 periodic timers, with periods APP_DLY_TIMER0_ms,
+APP_DLY_TIMER1_ms and APP_DLY_TIMER2_ms. Each timer is attached an LED (error.
+radio and sync). When you run the application, you should see the LEDs
+"counting".
+
+\author Pere Tuset <peretuset@openmote.com>, January 2015.
+*/
+
+#include "stdint.h"
+#include "stdio.h"
+
+#include "board.h"
+#include "debugpins.h"
+#include "leds.h"
+#include "scheduler.h"
+
+#include "../../../drivers/OpenMote-CC2538/adxl346.h"
+#include "../../../drivers/OpenMote-CC2538/max44009.h"
+#include "../../../drivers/OpenMote-CC2538/sht21.h"

```

Comment:
@peretuset, we can avoid including the full path to the sensor drivers, simply you could change the Sconscript.env of OpenMote-CC2538 adding 

os.path.join('#','drivers',buildEnv['board'])

in the CPPPATH